product_combination_exclude/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Graeme Gellatly
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Combination Exclude',
+    'summary': "Allows the specification of incompatible product "
++              "combinations when creating products",
```

Comment:
Looks like the copy/paste was from a git diff? There's a `+` at the beginning of this line